FlatCache/FlatCache.swift

```diff
@@ -136,5 +136,9 @@ public final class FlatCache {
         return storage[key] as? T
     }
 
+    public func clearCache() {
+        storage = [:]
+    }
```

Comment:
Also, Do we need to let the `listeners` know that cache is being cleared. Im not sure here